themes/classic/templates/layouts/layout-both-columns.tpl

```diff
@@ -23,7 +23,7 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  *}
 <!doctype html>
-<html lang="{$language.iso_code}">
+<html lang="{$language_code}">
```

Comment:
Is it the first layout containing this? I doubt as there are multiple ones

Reply:
As far as I know, the other layouts extend the `layout-both-columns.tpl`, so this change will apply to all of them.